lib/docker-toolbelt.coffee

```diff
@@ -42,8 +49,23 @@ createChainIdFromParent = (parent, dgsts) ->
 
 	return createChainIdFromParent(dgst, dgsts[1..])
 
+getDiffIds = (dkroot, driver, imageId) ->
+	[ hashType, hash ] = imageId.split(':')
```

Comment:
This could throw if `imageId` is not a string (which would be a bit of a surprise since it normally returns a promise)